client/python/tx_clock.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+from functools import total_ordering
+
+@total_ordering
+class TxClock(object):
+
+    # Bounds on TxClock values
+    MinValue = 0
+    # In Python, longs are actually unbounded, but we'll give TxClock a max
+    # TODO: How big do we want the max to be?
+    MaxValue = 2**64 - 1
+
+    # REQUIRES: Input time in seconds
+    def __init__(self, time=None, input_in_usecs=False):
+        self._input_in_usecs = input_in_usecs
+
+        # Default to current time
+        if (time == None):
```

Comment:
Convert the system time to a long in the constructor, and drop the epsilon compare below.